src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -529,7 +526,10 @@ class HMICapabilities {
   virtual const smart_objects::SmartObject* seat_location_capability()
       const = 0;
 
-  virtual void Init(resumption::LastState* last_state) = 0;
+  DEPRECATED
+  virtual void Init(resumption::LastState&) {}
```

Comment:
Need to properly write this function, empty body will not work properly

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
Function body is still empty.

Reply:
@jacobkeeler Please take a look https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r373481596